### PR TITLE
Add requirements for specifying Powerful Features

### PR DESCRIPTION
**Pull request by marcoscaceres**
Can link to this from the registry as part of the registration instructions...

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/permissions/pull/362.html" title="Last updated on Feb 14, 2022, 9:09 PM UTC (e2872a2)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/permissions/362/cbed850...e2872a2.html" title="Last updated on Feb 14, 2022, 9:09 PM UTC (e2872a2)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -966,7 +979,12 @@ <h3 id="screen-wake-lock">
         </p>
       </section>
     </section>
-    <section id="conformance"></section>
+    <section id="conformance">
+      <p>
+        Two class of products can claim conformance to this document: a [=user agent=] and a
```
Comment (miketaylr) [inline index.html:984]:
```suggestion
        Two classes of products can claim conformance to this document: a [=user agent=] and a
```

**Review by miketaylr — APPROVED** [review]
lgtm w/ typo

**Comment by miketaylr** [comment]
> lgtm w/ typo

I guess a tiny merge conflict to resolve as well.

**Comment by marcoscaceres** [comment]
Great! Will resolve in the morning.

**Comment by miketaylr** [comment]
Cool - if I remember I'll try to do it for ya in between meetings today.

File: index.html [hunk 2]
```diff
@@ -302,15 +302,28 @@ <h3>
...
-      <p class="advisement">
-        Please register newly specified [=powerful features=] in the
-        [[[powerful-feature-registry]]]. Doing so also provides this Working Group an opportunity
-        to provide feedback and check that integration with this specification is done effectively.
-      </p>
       <p>
-        Each <a>powerful feature</a> has the following permission-related algorithms and types.
-        When the defaults are not suitable for a particular [=powerful feature=], a specification
-        MAY override below algorithms and types below.
+        When a [=specification=] <dfn class="export">specifies a powerful feature</dfn> it:
```
Comment (miketaylr) [inline index.html:306]: Hm, where is `[=specification=]` supposed to be linking to?

Comment (miketaylr) [inline index.html:306]:
```suggestion
        When a specification <dfn class="export">specifies a powerful feature</dfn> it:
```

Comment (miketaylr) [reply index.html:306]: Going to assume that was an accident and merge - if it was supposed to do something, please send a new PR! 🙈

**Pull request merged**